compiler/modules/bir/bir.bal

```diff
@@ -92,6 +94,19 @@ public type FunctionSignature readonly & record {|
     SemType? restParamType = ();
 |};
 
+public type Region record {|
+    Label entry;
+    Label? exit = ();
+    Label? parent = ();
+    RegionType ty;
```

Comment:
Abbreviating `type` to `ty` here is not consistent with other names.

I suggest you use `RegionKind kind;`

Reply:
fixed